sdk/clientcore/core/src/main/java/io/clientcore/core/instrumentation/tracing/NoopSpan.java

```diff
@@ -3,6 +3,7 @@
 
 package io.clientcore.core.instrumentation.tracing;
 
+import io.clientcore.core.implementation.instrumentation.NoopInstrumentationContext;
```

Comment:
Any reason this import was added?

Reply:
`NoopInstrumentationContext` was moved from this package to `io.clientcore.core.implementation.instrumentation`